### PR TITLE
Localisation fixes Arabic/French/German

### PR DESCRIPTION
**Pull request by RedDevilus**
Minor fixes to languages. I will add the languages soon. Please don't close this yet.
People can still send me commit/ zipped file to add to this bundle.

### CODE REVIEW

File: locales/ar_SA/pcsx2_Main.po [hunk 1]
```diff
@@ -1175,33 +1170,31 @@ msgstr "إختر ملف لإعدادات %s"
 msgid "Settings"
 msgstr "الإعدادات"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:86
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:84
```
Comment (lightningterror) [inline locales/ar_SA/pcsx2_Main.po]: The new line changes seem incorrect to me.

Comment (RedDevilus) [reply locales/ar_SA/pcsx2_Main.po]: Maybe he used an outdated file? Just the files i got from Plubu in Discord. I'll correct it.

Comment (RedDevilus) [reply locales/ar_SA/pcsx2_Main.po]: That line says pcsx2/gui/Dialogs/FirstTimeWizard.cpp:48. If anyone can look into this please.

Comment (lightningterror) [reply locales/ar_SA/pcsx2_Main.po]: 86 is correct

File: locales/de_DE/pcsx2_Main.po [hunk 2]
```diff
@@ -1201,14 +1201,12 @@ msgid "Welcome to PCSX2!"
...
-#, fuzzy
 msgid "Configuration Guide"
-msgstr "Konfigurationsratgeber (online)"
+msgstr "Konfigurationsratgeber"
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:110
-#, fuzzy
 msgid "Readme / FAQ"
-msgstr "Readme / FAQ (Offline/PDF)"
+msgstr "Readme / FAQ "
```
Comment (pgert) [inline locales/de_DE/pcsx2_Main.po]: Remove the Space at the end.

File: locales/ar_SA/pcsx2_Main.po [hunk 3]
```diff
@@ -5,28 +5,29 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 1.0\n"
+"Project-Id-Version: PCSX2 1.6\n"
```
Comment (lightningterror) [inline locales/ar_SA/pcsx2_Main.po]: 1.6.0

Comment (RedDevilus) [reply locales/ar_SA/pcsx2_Main.po]: Thats weird. It doesnt say that with mines. Maybe i committed it to quickly.

Comment (RedDevilus) [reply locales/ar_SA/pcsx2_Main.po]: https://github.com/RedDevilus/pcsx2/blob/RedDevilus-ML-Localisations-3/locales/ar_SA/pcsx2_Main.po

File: locales/ar_SA/pcsx2_Main.po [hunk 4]
```diff
@@ -1157,21 +1153,20 @@ msgid ""
...
+"هذا هو الأساس الذي تستخدمه Sony  و هو مدعوم من قبل جميع الألعاب ونسخ الـ BIOS "
 
 #: pcsx2/gui/Dialogs/CreateMemoryCardDialog.cpp:260
 msgid ""
 "This memory card is required by PSX games. It is not compatible with PS2 "
 "games."
 msgstr ""
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:48
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:50
```
Comment (lightningterror) [inline locales/ar_SA/pcsx2_Main.po]: Forgot to revert this one.

File: locales/ar_SA/pcsx2_Main.po [hunk 5]
```diff
@@ -1157,21 +1153,20 @@ msgid ""
...
 #, c-format
 msgid "Select a folder for %s settings"
 msgstr "إختر ملف لإعدادات %s"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:48
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:50
```
Comment (lightningterror) [inline locales/ar_SA/pcsx2_Main.po]: Here too.

File: locales/ar_SA/pcsx2_Main.po (hunk 3 above)
Comment (lightningterror) [reply locales/ar_SA/pcsx2_Main.po]: Forgot the 0.

File: locales/de_DE/pcsx2_Main.po [hunk 6]
```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Project-Id-Version: PCSX2 1.6.0\n"
 "Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
 "POT-Creation-Date: 2020-02-10 17:29+0000\n"
-"PO-Revision-Date: 2020-02-04 19:21+0100\n"
-"Last-Translator: XXXBold\n"
+"PO-Revision-Date: 2020-02-12 01:46+0100\n"
+"Last-Translator: RedDevilus\n"
```
Comment (lightningterror) [inline locales/de_DE/pcsx2_Main.po]: It would be nice to also keep the recent translators as well.

File: locales/ar_SA/pcsx2_Main.po (hunk 4 above)
Comment (RedDevilus) [reply locales/ar_SA/pcsx2_Main.po]: 48 is that line?

**Comment by RedDevilus** [comment]
Arabic iconised .po and .mo has not been touched. I don't think we have time to translate that. Let it be 
turtleli update only for code updates i guess. I'll wait till 16 February for other languages (afterwards just review and merge)

File: locales/ar_SA/pcsx2_Main.po [hunk 7]
```diff
@@ -2355,8 +2351,8 @@ msgstr "إنشاء مستنسخ من هذا الكرت"
 
 #: pcsx2/gui/Panels/MemoryCardListPanel.cpp:489
 msgid ""
-"Convert this memory card to or from a folder memory card. Creates a "
-"duplicate of the current memory card in the new type.\n"
+"Convert this memory card to or from a folder memory card. Creates a duplicate "
```
Comment (lightningterror) [inline locales/ar_SA/pcsx2_Main.po]: Forgot to change this back.

**Comment by RedDevilus** [comment]
Hopefully this is the end of this never-ending story of the 1001 Nights of Arabic.

**Comment by RedDevilus** [comment]
You can merge this. I'll just make a new PR when needed and wouldnt be this messy.

**Comment by CaptainTearow** [comment]
Do you plan to provide an easy way to translate like with Weblate or Crowdin?
I'm french and this doesn't look good but I don't know how to help with this.
Like "Sauvegardes :" is replaced by Sauvegardes:"
But the first one is right not the second. In french we put Non-breaking space before a punctuation we do not put the punctuation with the word. The "correction" in this PR make this mistake a lot (but yes the simple . is with the word this is probably the only exception I can think of rn).
I hope this can help and if there is somewhere I can provide correction let me know.

**Comment by RedDevilus** [comment]
I forgot that rule existed in french. Je serais stupide. I'll fix it with the next PR.

**Pull request merged**